packages/core/content-manager/server/controllers/collection-types.js

```diff
@@ -181,6 +181,18 @@ module.exports = {
 
     const sanitizedBody = await sanitizeFn(body);
 
+    const entityAttributes = entity.attributes || {};
+
+    Object.keys(entityAttributes).forEach((attributeName) => {
+      if (entity.attributes[attributeName].type === 'media') {
+        if (entity[attributeName]) {
+          sanitizedBody[attributeName] = entity[attributeName];
+        } else {
+          sanitizedBody[attributeName] = null;
+        }
+      }
+    });
```

Comment:
This should take into account if the content-manager body request is trying to delete de media attribute. With this implementation it would always copy the media if the original entry does have some.

side note: this would only include the media attributes when the entry is cloned from the content-manager.
The cloning utility starts at the database layer. The ideal fix should be placed in there, but that would require much more work, so lets continue with this approach for now :)

Reply:
Sorry for the late response, okay I see what you are implying, so what's next? Should I start working on an ideal fix?